Rubberduck.Core/UI/About/AboutControl.xaml

```diff
@@ -177,31 +178,38 @@ Stack Overflow
 Andrew Jackson
 Andrew Zschetzsche
 Andrin Meier
+Anna Velcheva
```

Comment:
@AnnaVel @Imh0t3b @IpshitaC @NelsonVides @tommy9 - sorry it took so long to update this, thanks again for your work!